### DISCUSSION TITLE
Welcome to Festival2024 Discussions!

### DISCUSSION THREAD

**Discussion by julianpistorius** [resbazaz/festival2024#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to organize the 2024 festival. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by julianpistorius** [comment]
Hi! I have a software engineering background, and enjoy helping researchers with computation. I am a regular at @resbazaz/hacky-hour on Thursdays.

**Comment by jcoliver** [comment]
I am a recovering biologist, and my work is largely in training folks in computational and data literacy. I primarily lurk in Slack.

**Comment by rbartelme** [comment]
Jack of all trades, master of some. Ecology, agriculture, data science, bioinformatics, software engineering, devops, and sometimes I cosplay as a network sysadmin or mathematician. One of the @resbazaz/hacky-hour organizers with @julianpistorius.

**Comment by e-eight** [comment]
I am a physicist, currently roleplaying a HPC consultant at UA. I am stationed at Phoenix/Tempe.

**Comment by brandon-jernigan** [comment]
Hi! I’m Brandon, I studied biomedical engineering/informatics and I work at the UA Health Sciences Biorepository. I’m a regular at @resbazaz/hacky-hour, @resbazaz/coffee-and-code, and code commons

**Comment by hidyverse** [comment]
Hi I'm Heidi! I'm a new budding corporate clinical data scientist and a longtime fan of ResBaz.

**Comment by lwdozal** [comment]
Hi! I'm Laura, a ABD PhD student at the School of Information here at the UofA! Two year ResBaz instructor and long-time participant of the conference.
